src/flow/migrate/transactions.rs

```diff
@@ -36,12 +34,6 @@ fn count_base_ingress(
                 })
                 .sum();
 
-            // Domains containing a base node will get a single copy of each packet sent to it.
-            if graph[base].domain() == *domain {
-                assert_eq!(num_paths, 0);
-                num_paths = 1;
-            }
```

Comment:
This seems like a semantic change, no? Why is it okay to remove it?

Reply:
nvm -- found the moved check below.

Reply:
It was just moved, since it doesn't actually compute anything over the ingress nodes.
See https://github.com/mit-pdos/distributary/pull/41/files/56f572e445bf2517185fb689dbf4f37c75232f00#diff-5af68982bf87be5004d3320a768608a8R122